pennylane/ops/identity.py

```diff
@@ -112,6 +112,10 @@ def compute_matrix(n_wires=1):  # pylint: disable=arguments-differ
     def compute_sparse_matrix(n_wires=1):  # pylint: disable=arguments-differ
         return sparse.eye(int(2**n_wires), format="csr")
 
+    def matrix(self, wire_order=None):
+        n_wires = len(wire_order) if wire_order else len(self.wires)
+        return self.compute_matrix(n_wires=n_wires)
```

Comment:
Added this so it doesnt call `matrix_expand` in https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/operation.py#L802